developer-workflow/development-cycle.rst

```diff
@@ -77,10 +77,21 @@ releases; the terms are used interchangeably. These releases have a
 **micro version** number greater than zero.
 
 The only changes allowed to occur in a maintenance branch without debate are
-bug fixes.  Also, a general rule for maintenance branches is that compatibility
+bug fixes. Also, a general rule for maintenance branches is that compatibility
 must not be broken at any point between sibling micro releases (3.5.1, 3.5.2,
-etc.).  For both rules, only rare exceptions are accepted and **must** be
-discussed first.
+etc.). For both rules, only rare exceptions are accepted and **must** be
+discussed first. Among the rare exceptions to the rules of bug fixes and compatibility,
+backporting of documentation and tests is encouraged.
+
+Backporting serves dual purposes. First, it increases the visibility of
+documentation changes since most users refer to stable versions at
+`docs.python.org/3/ <https://docs.python.org/3/>`_ rather
+than the `docs.python.org/dev/ <https://docs.python.org/dev/>`_ documentation.
+Second, it minimizes conflicts for future bugfix backports.
+
+Backporting should target enhancements in documentation and tests for
+currently **stable** versions, specifically branches in a **bugfix** release cycle
+or higher.
```

Comment:
Personally I think this is too verbose and I would delete or cut significantly. But I'll wait for Victor or Hugo's thoughts before committing to a request.

Reply:
Maybe just removing that line 86-90?

Reply:
```suggestion
Back-porting changes reduces the risk of future conflicts.
For documentation, it increases the visibility of improvements,
since most readers access the `stable documentation <https://docs.python.org/3/>`__
rather than the `development documentation <https://docs.python.org/dev/>`__.
```